doc/doc-site/docs/about/about.md

```diff
@@ -18,12 +18,15 @@ If you are a developer, see below for emojicoin.fun Move package addresses!
 
 <!-- markdownlint-disable MD013 -->
 
-| Chain   | Explorer link     | Full address                                                         |
-| ------- | ----------------- | -------------------------------------------------------------------- |
-| Mainnet | [`0xface...cafe`] | `0xface729284ae5729100b3a9ad7f7cc025ea09739cd6e7252aff0beb53619cafe` |
-| Testnet | [`0x1abf...73b4`] | `0x1abfa4c5bb5f381b00719fc19e8e655cb2531904bf8f59309efd18eb081373b4` |
+| Chain   | Package             | Explorer link     | Full address                                                         |
+| ------- | ------------------- | ----------------- | -------------------------------------------------------------------- |
+| Mainnet | `emojicoin_dot_fun` | [`0xface...cafe`] | `0xface729284ae5729100b3a9ad7f7cc025ea09739cd6e7252aff0beb53619cafe` |
+| Mainnet | `arena`             | TBD               | TBD                                                                  |
+| Testnet | `emojicoin_dot fun` | [`0x1abf...73b4`] | `0x1abfa4c5bb5f381b00719fc19e8e655cb2531904bf8f59309efd18eb081373b4` |
+| Testnet | `arena`             | [`0x2e51...84c9`] | `0x2e51979739db25dc987bd24e1a968e45cca0e0daea7cae9121f68af93e8884c9` |
```

Comment:
```suggestion
| Mainnet | `emojicoin_arena`   | TBD               | TBD                                                                  |
| Testnet | `emojicoin_dot fun` | [`0x1abf...73b4`] | `0x1abfa4c5bb5f381b00719fc19e8e655cb2531904bf8f59309efd18eb081373b4` |
| Testnet | `emojicoin_arena`   | [`0x2e51...84c9`] | `0x2e51979739db25dc987bd24e1a968e45cca0e0daea7cae9121f68af93e8884c9` |
```
The named address is `emojicoin_arena`, see `Move.toml`: https://github.com/econia-labs/emojicoin-dot-fun/blob/0e05f9b67a578565efec01310216cd7881e29c44/src/move/emojicoin_arena/Move.toml#L2